src/lib/headers/SidebarHeader.js

```diff
@@ -64,7 +65,9 @@ SidebarWrapper.propTypes = {
 
 SidebarWrapper.defaultProps = {
   variant: LEFT_VARIANT,
-  children: ({ getRootProps }) => <div data-testid="sidebarHeader" {...getRootProps()} />
+  children: ({ getRootProps }) => (
```

Comment:
We are getting the following warning here:

```sh
/forks/react-calendar-timeline/src/lib/headers/SidebarHeader.js
  68:13  error  Component definition is missing display name   react/display-name
  68:16  error  'getRootProps' is missing in props validation  react/prop-types
```

What would you suggest to fix it? :)